utils/mariadb/Makefile

```diff
@@ -9,10 +9,11 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=mariadb
 PKG_VERSION:=10.2.19
-PKG_RELEASE:=2
+PKG_RELEASE:=3
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL := \
+	https://sources.openwrt.org \
```

Comment:
I don't think that you should add  sources.openwrt.org to the download site list.

It serves as a fallback option in any case, but should not be listed separately for each package.

Reply:
Hello Hannu,

OK :) I removed it.

Kind regards,
Seb